core/client/app/services/config.js

```diff
@@ -2,27 +2,24 @@ import Ember from 'ember';
 
 const {Service, _ProxyMixin, computed} = Ember;
 
-function isNumeric(num) {
+function _isNumeric(num) {
```

Comment:
@kevinansfield It literally just wraps the isNumeric from jQuery - so it's searchable. But :+1: for consistency with the other helpers.